apps/backend-mock/api/crypto/tokenStore.ts

```
@@ -0,0 +1,25 @@
+interface TokenBalances {
+  like: number;
+  usdt: number;
+}
+
+export let userTokenBalances: TokenBalances = {
+  like: 1000, // Initial "like" token balance
+  usdt: 500,   // Initial USDT balance
+};
+
+export const updateUserTokenBalance = (tokenType: 'like' | 'usdt', amount: number): void => {
+  if (userTokenBalances.hasOwnProperty(tokenType)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)